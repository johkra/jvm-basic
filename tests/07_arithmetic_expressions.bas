PRINT 3 + 2 * 12 / 3
PRINT 3 + 2 * 6^4 / 3
PRINT 3 + 2 * 6^4
PRINT 1 + 1 * 1 + 1
