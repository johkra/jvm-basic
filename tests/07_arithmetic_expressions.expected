11
867
3
